backend/src/main/kotlin/com/manage/crm/email/application/SendNotificationEmailUseCase.kt

```
@@ -114,38 +114,33 @@ class SendNotificationEmailUseCase(
-    private fun generateNotificationDto(targetUsers: Map<String, User>, notificationProperties: NotificationEmailTemplatePropertiesModel): List<SendEmailInDto> {
-        return targetUsers.keys
-            .mapNotNull { email ->
-                doGenerateContent(targetUsers, notificationProperties, email)
-                    ?.let { doMapToNotificationDto(email, notificationProperties, it) }
-            }
-            .toList()
+    private fun parseUserAttributes(user: User): Map<String, Any>? {
+        return try {
+            val typeRef = object : TypeReference<Map<String, Any>>() {}
+            objectMapper.readValue(user.userAttributes.value, typeRef)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)